src/app/ubs-admin/services/admin-orders.service.ts

```diff
@@ -0,0 +1,15 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AdminOrdersService {
+  backend = 'assets/orders.json';
```

Comment:
move this to mocks folder

Reply:
backend = 'assets/mocks/orders/orders.json';